src/funcs.c

```diff
@@ -1909,12 +1909,12 @@ void            ExecEnd (
 *F  FuncSetRecursionTrapInterval( <self>, <interval> )
 **
 */
-
+#include <stdio.h>
```

Comment:
BTW, the first line of the commit message "Ensure RecursionTrapInterval is big enough we can enter the break loop without hitting it" exceeds 70 chars and gets wrapped by GitHub. At the same time, I did not really understand it, and had to look at the content of the commit.

How about this:
```
Ensure RecursionTrapInterval is always at least 5

For smaller values, using the break loop becomes impossible
```